pkg/service/pipelinerun.go

```diff
@@ -346,11 +359,15 @@ func (s *service) ListPipelineRunsByRequester(ctx context.Context, req *pb.ListP
 		if err != nil {
 			return nil, fmt.Errorf("converting pipeline run: %w", err)
 		}
-		pbRun.RunnerId = runnerMap[run.TriggeredBy]
+		pbRun.RunnerId = userIDMap[run.TriggeredBy]
```

Comment:
nit: it seems we're maintaining the same code in 2 places. We might want to extract the common code to a function.

Reply:
I've thought of this but it will include an API call to mgmt private service. Did not come up with a good idea on this so I did not extract them yet.
Also I'm also considering adding a memcache package to cache the user IDs here.